charts/internal/shoot-system-components/charts/csi-alicloud/templates/crd-volumesnapshotclasses.yaml

```diff
@@ -1,80 +0,0 @@
-{{- if semverCompare ">= 1.17" .Values.kubernetesVersion }}
```

Comment:
Why you decided to remove these CRDs?

Reply:
I supposed they were not used. See [here](https://github.com/gardener/gardener-extension-provider-alicloud/blob/master/pkg/controller/controlplane/valuesprovider.go#L217). Did I make some mis-understanding?

Reply:
OK. BTW, how long do you plan to keep the legacy one?